app/Http/Controllers/Admin/ProductController.php

```diff
@@ -114,16 +114,14 @@ public function store(StoreProductRequest $request)
                 );
             };
 
-            foreach ($request->files as $files) {
-                foreach ($files as $file) {
-                    $img = uploadFile('files', config('path.PRODUCT_UPLOAD_PATH'), $request, $file);
-                    $this->productImageRepository->create(
-                        [
-                        'product_id' => $product->id,
-                        'path' => $img,
-                        ]
-                    );
-                }
+            foreach ($request->images as $file) {
+                $img = uploadFile('images', config('path.PRODUCT_UPLOAD_PATH'), $request, $file);
+                $this->productImageRepository->create(
+                    [
+                    'product_id' => $product->id,
+                    'path' => $img,
+                    ]
```

Comment:
```suggestion
                    [
                        'product_id' => $product->id,
                        'path' => $img,
                    ]
```
fix tương tự các chỗ khác nha